src/components/scene-preview-camera.js

```diff
@@ -54,6 +55,10 @@ AFRAME.registerComponent("scene-preview-camera", {
   tick2: function() {
     const hubsSystems = this.el.sceneEl.systems["hubs-systems"];
 
+    uiRoot = uiRoot || document.getElementById("ui-root");
+    const isGhost = uiRoot && uiRoot.firstChild && uiRoot.firstChild.classList.contains("isGhost");
```

Comment:
There's a lot of checking for this "isGhost" class. I'm a little worried about the perf of the number of calls to `contains` every frame.

Reply:
will only check for `isGhost` if the scene is not entered yet